yarn.lock

```diff
@@ -7175,6 +7206,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"keyv@npm:^5.3.1":
```

Comment:
General comment: It's interesting that cache-manger uses this. Keyv also [supports Redis](https://keyv.org/docs/storage-adapters/redis/) so maybe we could simplify this logic (in the future).